### PR TITLE
[Improvement-4971][alert|dao|service]Remove redundant PropertyUtils.class

### LINKED ISSUE #4971: [Improvement][common] Remove redundant PropertyUtils.class
We are flooded with too many PropertyUtils.class, in my opinion, there is no need, therefore, it is best to keep only the common-model.

### PR DESCRIPTION
**Pull request by Ts686**
## *Tips*
- *Thanks very much for contributing to Apache DolphinScheduler.*
- *Please review https://dolphinscheduler.apache.org/en-us/community/development/pull-request.html before opening a pull request.*

this closes #4971 

## What is the purpose of the pull request

*(For example: This pull request adds checkstyle plugin.)*

## Brief change log

*(for example:)*
  - *Add maven-checkstyle-plugin to root pom.xml*

## Verify this pull request

*(Please pick either of the following options)*

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

*(example:)*

  - *Added dolphinscheduler-dao tests for end-to-end.*
  - *Added CronUtilsTest to verify the change.*
  - *Manually verified the change by testing locally.*

### CODE REVIEW

**Comment by CalvinKirs** [comment]
I understand should be inited when the respective application starts

**Comment by Ts686** [comment]
> I understand should be inited when the respective application starts

Now,this commit is the common.properties file loaded in the static code module , the alert.properties file loaded when the Alert Server started. 

u mean just load the datasource.properties file when the master sever, worker server and api server started.

Is it right?

**Comment by CalvinKirs** [comment]
> > I understand should be inited when the respective application starts
> 
> Now,this commit is the common.properties file loaded in the static code module , the alert.properties file loaded when the Alert Server started.
> 
> u mean just load the datasource.properties file when the master sever, worker server and api server started.
> 
> Is it right?

I think it should be like this

**Comment by CalvinKirs** [comment]
However, you can do this too.

File: dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/PropertyUtils.java [hunk 1]
```diff
@@ -51,7 +49,13 @@ private PropertyUtils() {
...
+        loadPropertyFile(COMMON_PROPERTIES_PATH);
+    }
+
+    /**
+     * init properties
+     */
+    public static void loadPropertyFile(String... propertyFiles) {
         for (String fileName : propertyFiles) {
             InputStream fis = null;
             try {
```
Comment (CalvinKirs) [inline dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/PropertyUtils.java]: There may be concurrency issues

Comment (Ts686) [reply dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/PropertyUtils.java]: > There may be concurrency issues

u mean synchronized method will be better？

Comment (Ts686) [reply dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/PropertyUtils.java]: but this method invoked only through static code module or the application start  , I don't think it has concurrency issues，right ?

Comment (CalvinKirs) [reply dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/PropertyUtils.java]: Your own rightness should not be tied to the specific business.

Comment (CalvinKirs) [reply dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/PropertyUtils.java:61]: Your own rightness should not be tied to the specific business.

**Comment by CalvinKirs** [comment]
You don't seem to have modified it?

**Comment by Ts686** [comment]
> You don't seem to have modified it?

Yes, I'm still working.I'll modify it when I get home. haha  
The company has no external network.

**Review by CalvinKirs — APPROVED** [review]
LGTM

**Comment by CalvinKirs** [comment]
The way the master and worker read configuration files may also be changed to this way, gradually removing Spring's related dependencies.

**Review by lgcareer — APPROVED** [review]
+1

**Pull request merged**